### PR TITLE
Rename Pangolin Minichef Contracts

### PR DESCRIPTION
**Pull request by zeroecksYeti**
Fixed `harvest()` function in `Avax-<token>` contracts, updated contract names, variable names, and snowglobe names.

### CODE REVIEW

File: contracts/strategies/pangolin-minichef/strategy-png-avax-snob.sol [hunk 1]
```diff
@@ -2,12 +2,14 @@ pragma solidity ^0.6.7;
...
+contract StrategyPngAvaxSnobLp is StrategyPngMiniChefFarmBase {
     uint256 public _poolId = 22;
 
     // Token addresses
-    address public Png_AVAX_SNOB_lp = 0xa1C2c3B6b120cBd4Cec7D2371FFd4a931A134A32;
-    address public token1 = 0xC38f41A296A4493Ff429F1238e030924A1542e50;
+    address public png_avax_snob_lp = 0xa1C2c3B6b120cBd4Cec7D2371FFd4a931A134A32;
+    // snob already defined in strategy-base.sol:24
+    // should we add "override" specifier, or remove declaration from this contract?
+    // address public snob = 0xC38f41A296A4493Ff429F1238e030924A1542e50;
```
Comment (zeroecksYeti) [inline contracts/strategies/pangolin-minichef/strategy-png-avax-snob.sol:12]: @bigwampa please advise about this issue

**Comment by zeroecksYeti** [comment]
All contracts need to be tested still, I'm unable to at this time

File: contracts/strategies/pangolin-minichef/strategy-png-avax-ampl.sol [hunk 2]
```diff
@@ -18,92 +18,88 @@ contract StrategyPngAVAXAMPLMiniLp is StrategyPngMiniChefFarmBase {
...
             uint256 _keep = _png.mul(keep).div(keepMax);
-            uint256 _amount = _png.sub(_keep).div(2);
             if (_keep > 0) {
                 _takeFeePngToSnob(_keep);
             }
+
+            _png = IERC20(png).balanceOf(address(this));
+
             IERC20(png).safeApprove(pangolinRouter, 0);
             IERC20(png).safeApprove(pangolinRouter, _png.sub(_keep));
```
Comment (AbigailTCameron) [inline contracts/strategies/pangolin-minichef/strategy-png-avax-ampl.sol:46]: no need to subtract the keep because we find the new png balance at line 43

**Review by AbigailTCameron — APPROVED** [review]
Deleted duplicates. Test passed

**Review by AbigailTCameron — DISMISSED** [review]
tests passed!

**Comment by jomarip** [comment]
This seems duplicative of Pull Request #132

**Comment by jomarip** [comment]
@Ncookiez said this was not duplicative, there are some different changes.

**Review by bigwampa — APPROVED** [review]

**Pull request merged**